shared/database/task/task_event.py

```diff
@@ -51,6 +53,21 @@ def serialize(self):
             data['comment'] = self.comment.serialize()
         return data
 
+    @staticmethod
+    def get_last_task_comment(session, task_id, job_id, project_id):
+        latest_comment = session.query(TaskEvent).filter(
+            TaskEvent.task_id == task_id,
+            TaskEvent.job_id == job_id,
+            TaskEvent.project_id == project_id,
+            TaskEvent.event_type == 'comment'
+        ).order_by(desc(TaskEvent.time_created)).first()
+
+        if (latest_comment == None):
+            return ""
+
+        comment_display = session.query(DiscussionComment).filter(DiscussionComment.id == latest_comment.comment_id).first()
+        return comment_display.__dict__["content"]
```

Comment:
Correct me if I'm wrong, but I think here you con do just `comment_display.content` and you should get the same result maybe?

Reply:
Yeah, you are totally right, just fixed this